opi/plugins/yandex-browser.py

```diff
@@ -23,7 +23,7 @@ def run(cls, query):
 		opi.add_repo(
 			filename = option,
 			name = option,
-			url = f'https://repo.yandex.ru/yandex-browser/rpm/{release}/$basearch/',
+			url = 'https://repo.yandex.ru/yandex-browser/rpm/{release}/$basearch/',
```

Comment:
This unrelated change would break things. Format strings need to have the `f` prefix.
See https://docs.python.org/3/reference/lexical_analysis.html#f-strings